src/styles/blocks/catalog.scss

```diff
@@ -21,5 +21,11 @@
     &:not(:nth-child(4n)) {
       margin-right: 48px;
     }
+
+    &:hover {
+      transform: scale(120%);
```

Comment:
The scale property should have a value of 1.2 for a 20% increase. The current value of 120% actually scales the element by 120 times.